util/chplenv/chpl_3p_jemalloc_configs.py

```diff
@@ -21,10 +21,15 @@ def get_jemalloc_config_file():
     return config_file
 
 @memoize
-def get_link_args():
-    jemalloc_config = get_jemalloc_config_file()
-    libs = ['-ljemalloc']
-    if os.access(jemalloc_config, os.X_OK):
-        jemalloc_libs = run_command([jemalloc_config, '--libs'])
-        libs += jemalloc_libs.split()
-    return libs
+def get_link_args(target_mem):
+    if target_mem == 'jemalloc':
+        jemalloc_config = get_jemalloc_config_file()
+        libs = ['-ljemalloc']
+        if os.access(jemalloc_config, os.X_OK):
+            jemalloc_libs = run_command([jemalloc_config, '--libs'])
+            libs += jemalloc_libs.split()
+        return libs
+    elif target_mem == 'system':
+        return ['-ljemalloc']
```

Comment:
I see some Debian packages install .la files to /usr/lib and others install .pc files to /usr/lib/<arch>/pkgconfig. I think it would be better to worry about this issue once there is a jemalloc package needing -lrt on some system that we're trying to work with. That way, we can test it & I don't have to predict what the package maintainers will do.